hack/label-node.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+KUBECTL=${KUBECTL:-oc}
+NAMESPACE=${NAMESPACE:-"instaslice-system"}
+WEBHOOK_TIMEOUT=${WEBHOOK_TIMEOUT:-2m}
```

Comment:
`WEBHOOK_TIMEOUT` is declared but not used in the script.

Reply:
done